<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableViewAudio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>361</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Add">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>350</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="song_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="playAndStopSong">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelVolume">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>volume</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextSong">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>40</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevSong">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>40</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
   </widget>
   <widget class="QPushButton" name="offMusic">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>40</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeWindow">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>350</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QSlider" name="musicSlider">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>70</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="songTime">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="muteButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>51</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Mute</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>379</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
